contracts/Removal.sol

```diff
@@ -31,6 +31,21 @@ contract Removal is
     // todo: supplier name
   }
 
+  struct BatchMintRemovalsData {
+    address marketAddress;
+    bool list;
+  }
+
+  /**
+   * @notice Emitted on successful minting of a batch of removals.
+   */
+  event BatchMinted(uint256[] tokenIds);
+
+  /**
+   * @notice Emitted on successful listing of a batch of removals.
+   */
+  event BatchListed(uint256[] tokenIds);
```

Comment:
What was wrong with using the built-in events? Not opposed to using these just want to make sure added gas cost is defensible

Reply:
nothing wrong just didn't know they were there! using them now